src/bento/apis/vimeo-api.js

```diff
@@ -36,13 +36,22 @@ export const VIMEO_EVENTS = {
  * @return {string}
  */
 export function getVimeoIframeSrc(videoid, autoplay, doNotTrack) {
-  return addParamsToUrl(
-    `https://player.vimeo.com/video/${encodeURIComponent(videoid)}`,
-    {
-      'dnt': doNotTrack ? '1' : undefined,
-      'muted': autoplay ? '1' : undefined,
-    }
-  );
+  let identifier = encodeURIComponent(videoid);
+
+  const paramParts = videoid?.split('?h=') || [];
```

Comment:
I wonder if we should make this handle more url params other then just `h`. I think it'd be possible to generalize this to handle any url param.

Reply:
Hi @alanorozco, this is indeed a url from an embed code (provided by a client).

Reply:
In what context is the URL acquired? Do you have an example or two?

Reply:
@dethstrobe I agree, the single parameter splitting here is a little strange, especially as it is referred to as 'videoid' incoming.

@alanorozco The use case was a client dropping an embed url of a private Vimeo video into our app, which parses it for the url. Upon investigation, it seems the 'share link' url to the video will work, so perhaps our app can swap out one for the other instead.

I still think url param handling may be necessary for the amp-vimeo component in some capacity, as Vimeo allows the ability to pass some things like '?quality=720p'. In its current form, this PR is not currently fit for that purpose. Happy to take direction on this issue.